.github/workflows/test.yml

```diff
@@ -25,6 +27,11 @@ jobs:
         run: python -m pip wheel -w $HOME/wh .
       - name: install cmeel
         run: python -m pip install $HOME/wh/cmeel*.whl
+      - name: check compatible tags
+        shell: bash -l {0}
+        run: |
+          python3 -m pip debug --verbose
+
```

Comment:
Is this still required, or was it just a temporary debug ?

Reply:
I thought like this we can verify if the compatible tags become available again and then take off the workaround. But it's not necessary, I can delete it.

Reply:
Ok, then I guess we can just remove the first commit from the PR :)

Reply:
I removed the commit. Good to merge now